composer.json

```diff
@@ -27,7 +27,7 @@
         "php-amqplib/php-amqplib": "^2.0",
         "phpunit/phpunit": "^5.7.27 || 6.5.8 || ^7.1.2",
         "predis/predis": "^1.0",
-        "sensiolabs/security-checker": "^1.3",
+        "sensiolabs/security-checker": "*",
```

Comment:
We are not fans of wildcard constraints. They are generally slow to resolve, and any changes between the lowest shipped stable version and the latest version can lead to huge maintenance nightmares.

Since we know it works on `^1.3`, this should instead do `||` constraints, and the `.travis.yml` should be updated so we test against every major version in the constraint list. Alternatively, we can bump the minimum supported version of security-checker for the next minor release.

Reply:
I updated the version constraint.

I don't think updating `.travis.yml` is really required as we are mocking the expected response from `SensioLabs\Security\SecurityChecker::check()` and not calling directly.